### PR TITLE
Workflow Automated Test - LRQA-69296 | master 

### PR DESCRIPTION
**Pull request by alessandromalves**
https://issues.liferay.com/browse/LRQA-69296

### CODE REVIEW

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by alessandromalves** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -244,12 +244,62 @@ definition {
...
+		
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "Assign to Me");
+
+		Workflow.confirmWorkflowAction();
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "Approve");
+
+		Workflow.confirmWorkflowAction();
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowDocumentsAndMedia.testcase]: You can use macros for this

**Comment by lucasperj** [comment]
combined and sent on [PR#76](https://github.com/lucasperj/liferay-portal/pull/76)

**Pull request closed without merging**